lib/theme_check/template.rb

```diff
@@ -58,6 +66,17 @@ def root
       parse.root
     end
 
+    def replace_line(line)
```

Comment:
This is confusing. According to the method name, I would expect this method to actually replace the line.

Either I would expose an `updated!` method, or I would rename this method to make it clearer it is not doing the replacement.

W/ the `updated!` method:
```
line = template.full_line(line_number)
line.insert(...)
template.updated!
```

Reply:
that's a really good idea. I wanted to make use of the destructive nature of insert to update full line without having to duplicate the string. But was stuck at where the call `@updated = true` should reside. Having a `update!` method would do the trick